parsely/src/main/java/com/parsely/parselyandroid/EventsBuilder.java

```diff
@@ -96,90 +82,4 @@ Map<String, Object> buildEvent(
         return event;
     }
 
-    /**
-     * Collect device-specific info.
-     * <p>
-     * Collects info about the device and user to use in Parsely events.
-     */
-    private Map<String, String> collectDeviceInfo(@Nullable final String adKey) {
-        Map<String, String> dInfo = new HashMap<>();
-
-        // TODO: screen dimensions (maybe?)
-        PLog("adkey is: %s, uuid is %s", adKey, getSiteUuid());
-        final String uuid = (adKey != null) ? adKey : getSiteUuid();
-        dInfo.put("parsely_site_uuid", uuid);
-        dInfo.put("manufacturer", android.os.Build.MANUFACTURER);
-        dInfo.put("os", "android");
-        dInfo.put("os_version", String.format("%d", android.os.Build.VERSION.SDK_INT));
-
-        // FIXME: Not passed in event or used anywhere else.
-        CharSequence txt = context.getPackageManager().getApplicationLabel(context.getApplicationInfo());
-        dInfo.put("appname", txt.toString());
```

Comment:
Was this intentionally removed? If so, that's OK - just want to make sure it's not an oversight.